examples/with-next-i18next/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "private": true,
+  "name": "my-nextron-app",
+  "description": "My application description",
+  "version": "1.0.0",
+  "author": "Yoshihide Shiono <shiono.yoshihide@gmail.com>",
+  "contributors": [
+    "Gregor Adams <greg@pixelass.com>"
+  ],
+  "main": "app/background.js",
+  "scripts": {
+    "dev": "nextron",
+    "build": "nextron build",
+    "postinstall": "electron-builder install-app-deps"
+  },
+  "dependencies": {
+    "electron-serve": "^1.3.0",
+    "electron-store": "^8.1.0"
+  },
+  "devDependencies": {
+    "electron": "29.0.0-beta.4",
+    "electron-builder": "^24.9.1",
+    "next": "^13.5.6",
+    "next-i18next": "15.2.0",
```

Comment:
[MUST] Missing dependencies: `react-i18next` `i18next`

I'll add these deps after merge this PR :)